build/charts/antrea/crds/clustergroup.yaml

```diff
@@ -372,15 +251,6 @@ spec:
                         type: string
       subresources:
         status: { }
-  conversion:
-    strategy: Webhook
-    webhook:
-      conversionReviewVersions: ["v1", "v1beta1"]
-      clientConfig:
-        service:
-          name: "antrea"
-          namespace: "kube-system"
-          path: "/convert/clustergroup"
```

Comment:
Since the webhook is deleted, we should delete its "served-by: antrea-controlelr" label to stop populating CABundle for it.

Reply:
Removed the label.